sonarts-core/src/utils/navigation.ts

```diff
@@ -18,21 +18,14 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import * as ts from "typescript";
+import * as nodes from "./nodes";
 
 const Kind = ts.SyntaxKind;
 
 export function keyword(node: ts.BreakOrContinueStatement | ts.ThrowStatement | ts.ReturnStatement): ts.Node {
   return node.getFirstToken();
 }
 
-export function isAssignmentKind(kind: ts.SyntaxKind) {
```

Comment:
@stas-vilchik In this file there are 2 more functions which could be moved to `nodes`: `is` and `isToken`